### PR TITLE
Fetch address data only when tx becomes confirmed

### PR DESCRIPTION
**Pull request by nop33**
Closes https://github.com/alephium/mobile-wallet/issues/239
Closes https://github.com/alephium/mobile-wallet/issues/245

### CODE REVIEW

File: src/api/addresses.ts [hunk 1]
```diff
@@ -62,13 +62,11 @@ export const fetchAddressesTransactions = async (
   const results = []
 
   for (const addressHash of addressHashes) {
-    const txNumber = await client.explorer.addresses.getAddressesAddressTotalTransactions(addressHash)
```
Comment (nop33) [inline src/api/addresses.ts:65]: I removed this extra call since we don't use this anywhere.

**Comment by nop33** [comment]
@mvaivre This should improve the experience, could you please confirm that this fix addresses https://github.com/alephium/mobile-wallet/issues/239 ?

Thanks 🙏

**Comment by mvaivre** [comment]
Just tested it: it's so much better 👏 Great job! 👏

**Comment by mvaivre** [comment]
image

Few details I noticed:

1) Let's add some margin between the 2. Let's make the area with proposed words standalone, with round corners and reduce its height.
2) The animated button has a weird diamond shape
3) The bottom margin is too big

**Comment by mvaivre** [comment]
I made some changes to the import seed screen layout, please check it out!

There's just the scan button shape left to fix.

File: src/components/buttons/Button.tsx [hunk 2]
```diff
@@ -28,7 +29,7 @@ import { BORDER_RADIUS } from '~/style/globalStyle'
 export interface ButtonProps extends PressableProps {
   title?: string
   type?: 'primary' | 'secondary' | 'transparent' | 'tint'
-  variant?: 'default' | 'contrast' | 'accent' | 'valid' | 'alert'
+  variant?: 'default' | 'contrast' | 'accent' | 'valid' | 'alert' | 'highlight'
```
Comment (mvaivre) [inline src/components/buttons/Button.tsx:32]: Good initiative! But with this new prop we should maybe open an issue to one day get rid of the standalone `HighlightButton` which may be confusing... WDYT?

**Review by mvaivre — COMMENTED** [review]
Code looks good to me!

File: src/components/buttons/Button.tsx (hunk 2 above)
Comment (nop33) [reply src/components/buttons/Button.tsx:32]: Agreed

Comment (nop33) [reply src/components/buttons/Button.tsx:32]: https://github.com/alephium/mobile-wallet/issues/253

**Review by mvaivre — APPROVED** [review]
Annnnd, this LGTM!

(PS: while testing thoroughly, I noticed a couple of more things but not related to this PR, and created related issues)

**Pull request merged**